Add Boost versioning tutorial (#189)
microsoft/vcpkg-docs: vcpkg/consume/boost-versions.md

---
title: "Tutorial: Install a specific version of the Boost libraries using registry baselines"
description: Learn to install specific version of Boost packages using a baseline.
author: vicroms
ms.author: viromer
ms.prod: vcpkg
ms.date: 10/24/2023
ms.topic: tutorial

> #CustomerIntent: As a beginner vcpkg user, I want to install dependencies from multiple vcpkg registries

Author: ```suggestion
#CustomerIntent: As a beginner vcpkg user, I want to install a specific Boost version set
```

---
# Tutorial: Install a specific version of the Boost libraries using registry baselines"

> vcpkg provides the Boost set of libraries as individual packages as a

Reviewer: ```suggestion
vcpkg provides the `Boost` set of libraries as individual packages as a
```

> vcpkg provides the Boost set of libraries as individual packages as a

Reviewer: ```suggestion
vcpkg provides the `Boost` set of libraries as individual packages as a
```

convenience for users that depend only on a limited subset of Boost packages.

> However, this approach has a tradeoff. Users that want to lock a specific

Reviewer: ```suggestion
However, this approach has a tradeoff. Users who want to lock a specific
```
